src/set_theory/cardinal/basic.lean

```diff
@@ -924,18 +924,25 @@ theorem aleph_0_le {c : cardinal} : ℵ₀ ≤ c ↔ ∀ n : ℕ, ↑n ≤ c :=
   exact (nat.lt_succ_self _).not_le (nat_cast_le.1 (h (n+1)))
 end⟩
 
+theorem mk_eq_nat_iff {α : Type u} {n : ℕ} : #α = n ↔ nonempty (α ≃ fin n) :=
+begin
+  rw [← lift_mk_fin, ← mk_ulift, cardinal.eq],
+  exact nonempty.congr (λ e, e.trans equiv.ulift) (λ e, e.trans equiv.ulift.symm)
+end
+
+theorem lt_aleph_0_iff_finite {α : Type u} : #α < ℵ₀ ↔ finite α :=
+by simp only [lt_aleph_0, mk_eq_nat_iff, finite_iff_exists_equiv_fin]
+
 theorem lt_aleph_0_iff_fintype {α : Type u} : #α < ℵ₀ ↔ nonempty (fintype α) :=
```

Comment:
Surely there's no longer a need for this?

Reply:
Yes but it is used here and there, so I'll postpone its removal to another PR.

Reply:
I'd argue that removing this theorem is in scope, but since the PR is already delegated, you can do it later.

Reply:
@vihdzp The way I see it, while a PR author is free to do in-scope work to extend a PR, we shouldn't require a PR author to do this work unless it's somehow essential. As a reviewer, a way to make sure this sort of optional in-scope work makes it into a PR is to be willing to put in the work yourself and make it easy for the author to include the changes. (Or you keep it in mind and do a followup PR yourself.)

With this lemma, I figured Yury left it in because it'd be easier review-wise to remove it in a separate PR -- maybe as a reviewer I should have flagged it (like you did) to make sure that this was the case.

Reply:
This lemma is used in some `noncomputable` fintype instances. I think that we should review these instances and replace some of them with `[finite _]` instances.